hieradata/production.yaml

```diff
@@ -102,6 +102,9 @@ govuk_jenkins::job::network_config_deploy::environments:
   - 'carrenza-production-dr'
   - 'skyscape-production'
 
+govuk_jenkins::job::content_performance_manager:
```

Comment:
Thanks, splitting it was my attempt at avoiding a long line. I've changed it now to be as you posted.